pyproject.toml

```diff
@@ -5,9 +5,10 @@ requires = [
     "ninja",
     "cmake>=3.12",
     "pybind11>=2.10.4",
-    "numpy",
+    "numpy >= 2",
```

Comment:
Nitpick: `numpy>=2` as others above?

Reply:
Ah -- we build against 2.x for 1.x and 2.x compat. I can remove the spaces.